pkg/test/resourcefixture/testdata/basic/pubsub/v1beta1/pubsubtopic/_generated_object_pubsubtopic.golden.yaml

```diff
@@ -16,6 +16,30 @@ metadata:
   name: pubsubtopic-sample-${uniqueId}
   namespace: ${uniqueId}
 spec:
```

Comment:
This golden object change is due to the state-into-spec:merge in pubsubtopic. I would expect to see they are removed once we change the annotation's default to `absent`